docs/UserGuide.md

```diff
@@ -279,14 +284,25 @@ _Details coming soon ..._
 
 **Q**: How do I transfer my quiz attempt scores to other computers?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file
-that contains the data of your previous WeebLingo home folder.
+that contains the data of your previous Weeblingo home folder.
 
 **Q**: Can I add my customized flashcards?<br>
 **A**: Sorry, the current version of the application does not allow self-defined flashcards to be added. However, if you
 are advanced and know how to manipulate the storage files, you can do so if you want. However, the application assumes
 a certain format for each data field, thus if you modified the data wrongly, the app will use the fallback option, which
 is to load the storage file with built-in default flashcards and tags.
 
+**Q**: Why can't I delete some tags?<br>
+**A**: Default tags are provided by the application to aid in the learning process and guide you along the way. You can,
+however, add your own tags to any flashcard. The deletion of tags exists is to complement this user tagging function, 
```

Comment:
small typo. The deletion of tags exists to complement this user tagging function,

Reply:
o right thanks haha